scripts/build_renewable_profiles.py

```diff
@@ -217,6 +217,22 @@
 GEBCO_CRS = "EPSG:4326"
 
 
+def check_cutout_match(cutout, geodf):
+    cutout_box = box(*cutout.bounds)
+    region_box = box(*regions.total_bounds)
+
+    assert not region_box.intersection(cutout_box).is_empty, (
+        "The requester region is completely out cutout area.\n\r"
+        "Check please the provided cutout. More details on cutout generation are available in docs:\n\r"
+        "https://pypsa-earth.readthedocs.io/en/latest/tutorial.html#adjust-the-model-configuration \n\r"
+    )
+
+    if not region_box.covered_by(cutout_box):
+        logger.warning(
+            "Weather data does not fully cover the requester region. It's recommended to check the provided cutout. More details are provided in https://pypsa-earth.readthedocs.io/en/latest/tutorial.html#adjust-the-model-configuration"
```

Comment:
Thanks katia! :D
The code is ready to merge:)
I noticed this line that seemed a bit long; I like the division of the line in multiple lines as done above. That should make the coding fit better into the screen.
Could be nice to revise and the PR will be merged thereafter if you agree :)

Reply:
Thank a lot for the catch, Davide!
Fixed :D